### DISCUSSION TITLE
How to print a hot loader indicator in the browser?

### DISCUSSION THREAD

**Discussion by azabraao** [webpack/webpack#16344] [Q&A]
So we can enable hot loading and even see the progress in the browser console by setting `devServer.client.progress` as `true`. But how can I build a loading animation as we see in NextJS for when a modification is made?

**Comment by alexander-akait** [comment]
Can you explain (screenshot/video/etc)?

**Reply by azabraao** [reply]
@alexander-akait sure:

NextJS has this 'build watcher' that provides a visual indication to the developer when the code is rebuilding. I'm trying to achieve that with webpack 🤔 

<img width="1440" alt="image" src="https://user-images.githubusercontent.com/17601387/194365373-79b85d53-752f-4ba8-a2cf-024844ac5a2f.png">

**Reply by alexander-akait** [reply]
hm, we already have this for dev server, maybe I missed something?

**Reply by azabraao** [reply]
@alexander-akait could you point me in the direction where I would find how to implement it? I really didn't find it.

**Reply by alexander-akait** [reply]
We have this https://github.com/webpack/webpack-dev-server/blob/master/client-src/index.js#L180 and https://github.com/webpack/webpack-dev-server/blob/master/client-src/index.js#L133
